crane/views/v1.py

```diff
@@ -110,6 +112,30 @@ def repo_tags(repo_id):
     return repository.get_tags_for_repo(repo_id)
 
 
+@section.route('/repositories/<path:repo_id>/tags/latest')
```

Comment:
Is there a reason why the tag is hardcoded to `latest` instead of being a path param?

cc @mhrivnak